Makefile.am

```diff
@@ -12,6 +12,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = \
 	$(NULL)
 
 dist_systemdunit_DATA = \
+	eos-chown-avahi-helper-to-writers.service \
 	eos-companion-app.service \
 	eos-companion-app.socket \
```

Comment:
The files you remove are referenced here

Reply:
Agh, forgot to add my new changes 😅